flink-runtime/src/main/java/org/apache/flink/streaming/runtime/operators/sink/committables/CheckpointCommittableManagerImpl.java

```diff
@@ -117,7 +126,7 @@ SubtaskCommittableManager<CommT> getSubtaskCommittableManager(int subtaskId) {
     }
 
     @Override
-    public CommittableSummary<CommT> getSummary() {
+    public CommittableSummary<CommT> getSummary(int subtaskId, int numberOfSubtasks) {
         return new CommittableSummary<>(
                 subtaskId,
                 numberOfSubtasks,
```

Comment:
Something looks strange here. We should rename either the field `numberOfSubtasks` or the method parameter `numberOfSubtasks` to distinguish the number of sub tasks on receival and emission.